QUARKUS-279.md

```diff
@@ -0,0 +1,42 @@
+# QUARKUS-278 - Test and verify Quarkus HTTP/2 support
+
+JIRA link: https://issues.redhat.com/browse/QUARKUS-279
+
+Coverage should ensure full HTTP/2 support from client and server side.
+For native mode similar test coverage as for JVM mode needs to be ensured.
+
+## Scope of the testing
+Quarkus support by default HTTP/2 since version 1.4 (https://quarkus.io/blog/quarkus-1-4-final-released/), so the following action were taken to ensure test coverage:
+
+Client Side:
+
+- Setup some scenarios in order to cover `GRPC` 
+- Consume an HTTP/2 endpoint through `quarkus-rest-client`
+- Consume an HTTP/2 endpoint through `quarkus-rest-client-mutiny`
+
+Server Side:
+- Verify that http/2 is set as default HTTP protocol
+- Consume and endpoint developed with `quarkus-resteasy-jsonb` through HTTP/2 Client (Vertx HTTP Client)
+
+### Impact on testsuites and testing automation:
+
+
+### Impact on resources:
+ - More services will be running on OpenShift cluster, impact on CPU/memory should be minimal
```

Comment:
How many modules do you expect to be added into OpenShift TS?

In other words also asking if all the above can be covered in one module. Build + deployment time is the largest time portion from the overall execution time.

Execution time implication will be lower if you are able to limit number of new modules

Reply:
Yes, I think that this TP could be done in just one:

Have look this preview: https://github.com/quarkus-qe/quarkus-openshift-test-suite/pull/146

An existing module was reused (Http), I think that make sense.